store/src/test/java/com/zextras/mailbox/LikeXmlJettyServerTest.java

```diff
@@ -0,0 +1,59 @@
+// SPDX-FileCopyrightText: 2024 Zextras <https://www.zextras.com>
+//
+// SPDX-License-Identifier: AGPL-3.0-only
+
+package com.zextras.mailbox;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import com.zextras.mailbox.LikeXmlJettyServer.Builder;
+import com.zextras.mailbox.LikeXmlJettyServer.InstantiationException;
+import com.zimbra.cs.account.Config;
+import org.eclipse.jetty.server.Connector;
+import org.eclipse.jetty.server.Server;
+import org.junit.jupiter.api.Test;
+
+class LikeXmlJettyServerTest {
+
+  @Test
+  void shouldCreateJettyServer() throws InstantiationException {
```

Comment:
Hi @frisonisland , this test is big and generic. We could divide it by groups of assertions: a test for userConnector, another for adminConnector, etc. and centralize the mock setup part.

Reply:
Do you mean having different tests with same setup but different assert?
Like:
```suggestion
  @BeforeEach
  void setUp() {
    config = mock(Config.class);
    localServer = mock(com.zimbra.cs.account.Server.class);
    when(config.getHttpNumThreads()).thenReturn(20);
    when(config.getMailboxdSSLProtocols()).thenReturn(new String[] {"TLS1.2"});
    when(config.getSSLExcludeCipherSuites()).thenReturn(new String[] {"^TLS_RSA_.*"});
    when(config.getSSLIncludeCipherSuites()).thenReturn(new String[] {});
    when(config.getHttpConnectorMaxIdleTimeMillis()).thenReturn(3000);
  }

  @Test
  void shouldCreateJettyServer_withUserConnector() throws InstantiationException {
    final Server server = new Builder(config, localServer).build();
    
    final Connector[] connectors = server.getConnectors();
    final Connector userConnector = connectors[0];
    assertEquals("http/1.1", userConnector.getProtocols().get(0));
    assertEquals(3000, userConnector.getIdleTimeout());
  }
```

Reply:
Applied similar changes